spec/requests/regions_spec.rb

```diff
@@ -40,4 +40,112 @@
       "id"   => region.id.to_s
     )
   end
+
+  describe "/api/regions/:id/settings" do
+    # let(:server) { FactoryGirl.create(:miq_server) }
+    # let(:region) { FactoryGirl.create(:miq_region, :region => FactoryGirl.create(:miq_region, :region => ApplicationRecord.my_region_number)) }
+    let(:region) { @region }
+
+    let(:original_timeout) { region.settings_for_resource[:api][:authentication_timeout] }
+    let(:super_admin) { FactoryGirl.create(:user, :role => 'super_administrator', :userid => 'alice', :password => 'alicepassword') }
+
+    # before do
+    #   allow(MiqServer).to receive(:in_region).with(ApplicationRecord.my_region_number).and_return(MiqServer.where(:id => server.id))
+    # end
```

Comment:
@imtayadeway could you remove the commented code above ( & lines 45-46) and repush ?  unless you needed this for reference 😉